src/plugins/socketmon/private.h

```diff
@@ -323,6 +292,7 @@ struct udp_endpoint_win10_x64
     uint16_t port;
     addr_t localaddr; // local_address_win10_udp_x64
 } __attribute__ ((packed));
+using inetaf_win10_x64 = inetaf_win81_x64;
```

Comment:
Place this after the declaration of `struct inetaf_win81_x64`, not down here.

Reply:
OK.